docs/builtins/Keysets/keys-2.md

```diff
@@ -0,0 +1,33 @@
+## keys-2
+
+Use `keys-2` as a keyset predicate function to determine if at least two keys are matched in the keyset.
+
+### Basic syntax
+
+To use `keys-2` to check if at least two keys are matched in a keyset, use the following syntax:
+
+keys-2 count matched
```

Comment:
````suggestion
`(keys-2 count matched)`
````
also should this be a more involved example showing a full `enforce-keyset` expression with `keys-2` in a context?